src/cmd/linuxkit/gcp.go

```diff
@@ -185,7 +191,7 @@ func (g GCPClient) DeleteImage(name string) error {
 }
 
 // CreateInstance creates and starts an instance on GCP
-func (g GCPClient) CreateInstance(name, image, zone, machineType string, disks Disks, data *string, nested, replace bool) error {
+func (g GCPClient) CreateInstance(name, image, zone, machineType string, disks Disks, data *string, nested, vtpm, replace bool) error {
```

Comment:
At some point, we should replace all of these vars with a struct or variadic opts funcs. But not in this PR.